### PR TITLE
Release 0.5.14

### PR DESCRIPTION
**Pull request by mpgxvii**
- Bumped dependencies
- Updates topics endpoint resource type
- Updates authentication filter

### CODE REVIEW

File: src/main/kotlin/org/radarbase/gateway/filter/KafkaTopicsAuthFilter.kt [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+import jakarta.ws.rs.container.DynamicFeature
+import jakarta.ws.rs.container.ResourceInfo
+import jakarta.ws.rs.core.Context
+import jakarta.ws.rs.core.FeatureContext
+import org.radarbase.gateway.resource.KafkaTopics
+import org.radarbase.jersey.auth.filter.AuthenticationFilter
+
+class KafkaTopicsAuthFilter(
+    @Context private val authFilter: AuthenticationFilter,
```
Comment (mpgxvii) [inline src/main/kotlin/org/radarbase/gateway/filter/KafkaTopicsAuthFilter.kt]: @blootsvoets The `authFilter` here seems to not get resolved causing a NullPointerException. I couldn't figure out why.

**Review by blootsvoets — APPROVED** [review]
LGTM, thanks for initiating a PR

File: src/main/kotlin/org/radarbase/gateway/filter/KafkaTopicsAuthFilter.kt (hunk 1 above)
Comment (blootsvoets) [reply src/main/kotlin/org/radarbase/gateway/filter/KafkaTopicsAuthFilter.kt]: Indeed, I meant to return to this issue. Fixed in ec76b12.

Comment (mpgxvii) [reply src/main/kotlin/org/radarbase/gateway/filter/KafkaTopicsAuthFilter.kt]: Thank you!

**Pull request merged**